app/containers/AskQuestion/validate.js

```diff
@@ -0,0 +1,11 @@
+import messages from './messages';
```

Comment:
We have almost the same validations in other places. Can we store locations somewhere where we can reuse them (e.g. utils) and use them for different components?